cfpb/erap-data-processing
# Add entry for Port St. Lucie in city / county map

Add entry to county map to handle new city program in Port St. Lucie, Florida.

## Additions

- New entry for Port St. Lucie in Florida city / county map


## Testing

1. Check out branch and run script.
2. Verify that correct county is added to Port St. Lucie entry in output file.
